NuKeeper.Abstractions/Configuration/ISettingsContainer.cs

```diff
@@ -0,0 +1,17 @@
+using NuKeeper.Abstractions.Inspections.Files;
+
+namespace NuKeeper.Abstractions.Configuration
+{
+    public interface ISettingsContainer
```

Comment:
Shouldn't need an interface for a settings object should just be passed into things